include/BBEditor.h

```diff
@@ -86,6 +86,7 @@ public slots:
 	void removeSteps();
 	void addSampleTrack();
 	void addAutomationTrack();
+	void cloneBBTrackPattern();
```

Comment:
```suggestion
	void clonePattern();
```
Because we're in BBEditor, you can drop the "BBTrack" part of this function name.